pipelines/install/metadata.obsah.yaml

```diff
@@ -0,0 +1,18 @@
+help: |
+  Run an install pipeline
+variables:
+  forklift_state:
+    parameter: --state
+    help: Forklift state to ensure
+    # TODO: up/rebuild/destroy
+  pipeline_os:
+    parameter: --os
+    help: Operating system to install, like centos7, debian10 or ubuntu1804
```

Comment:
Are these the only operating systems that can be used? Can all the pipelines be run on any of the oses? Is it something more that the user would know when creating the pipeline?

Reply:
It maps to the `pipeline_os` variable which maps back to the base boxes. Of course it doesn't magically work so we will need packages for the OS. So you can use centos6, it will fail during provisioning since there's no packages. We'll also add a centos8 box at some point.